components/canvas/NodeTooltip.tsx

```diff
@@ -9,24 +15,56 @@ type NodeTooltipContainerProps = {
   isVisible?: boolean;
   position?: Position;
   style?: NodeToolbarProps["style"];
+  isEditing?: boolean;
+  nodeRef?: RefObject<HTMLDivElement>;
 };
 
 export const NodeTooltipContainer = ({
   children,
   isVisible,
   position,
```

Comment:
This unused prop can be removed from here and in the NodeTooltipContainerProps type